### PR TITLE
Add clojure Pedestal web framework

### PR DESCRIPTION
**Pull request by gicornachini**
Add [Pedestal](https://github.com/pedestal/pedestal) web framework.

| HTTP   | Route       | Status code | Response body         | Done? |
| ------ | ----------- | ----------- | --------------------- | ------- |
| `GET`  | `/`         | `200`       | **Empty**             | ✅ |
| `GET`  | `/user/:id` | `200`       | **id** given as param | ✅ |
| `POST` | `/user`     | `200`       | **Empty** | ✅ |

### CODE REVIEW

**Comment by waghanza** [comment]
Hey,

Thanks @gicornachini for this PR.

This will include `pedestal` for next run, on Monday.

@hlship Are you OK with such a thing ?

Regards,

**Comment by waghanza** [comment]
can you provide an example @hlship ?

I am pretty confused about JVM dialects with the version on jdk to use
https://hub.docker.com/_/clojure/tags contains a lot of, and not sure which jdk could be supported by clojure

File: clojure/pedestal/config.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+image: clojure:openjdk-11-lein-slim-buster
+
+build:
+  - lein uberjar
+
+binaries:
+  - target/the-benchmarker-pedestal-0.0.1-SNAPSHOT-standalone.jar
+
+command: java -Xms2G -server -Xmx2G -XX:+UseStringDeduplication -XX:+UseParallelGC -Dvertx.disableMetrics=true -Dvertx.threadChecks=false -Dvertx.disableContextTimings=true -Dvertx.disableTCCL=true -Dvertx.disableH2c=true -Dvertx.disableWebsockets=true -Dvertx.disableHttpHeadersValidation=true -Dvertx.flashPolicyHandler=false -Djava.net.preferIPv4Stack=true -jar target/the-benchmarker-pedestal-0.0.1-SNAPSHOT-standalone.jar 3000
```
Comment (waghanza) [inline clojure/pedestal/config.yaml:13]:
```suggestion
  image: clojure:openjdk-11-lein-slim-buster

  build:
    - lein uberjar

  binaries:
    - target/the-benchmarker-pedestal-0.0.1-SNAPSHOT-standalone.jar

  command: java -Xms2G -server -Xmx2G -XX:+UseStringDeduplication -XX:+UseParallelGC -Dvertx.disableMetrics=true -Dvertx.threadChecks=false -Dvertx.disableContextTimings=true -Dvertx.disableTCCL=true -Dvertx.disableH2c=true -Dvertx.disableWebsockets=true -Dvertx.disableHttpHeadersValidation=true -Dvertx.flashPolicyHandler=false -Djava.net.preferIPv4Stack=true -jar target/the-benchmarker-pedestal-0.0.1-SNAPSHOT-standalone.jar 3000
```

**Review by waghanza — CHANGES_REQUESTED** [review]

**Comment by gicornachini** [comment]
@waghanza Now I hope it will work, at least locally is running correctly with the docker container.

image

**Pull request merged**